sdcard/rootfs/kube-systemd/etc/kubernetes/dropins/docker-overlay.conf

```diff
@@ -7,7 +7,8 @@ EnvironmentFile=/etc/kubernetes/k8s.conf
 ExecStart=
 ExecStart=/usr/bin/docker daemon \
     -H unix:///var/run/docker.sock \
-    -s ${DOCKER_STORAGE_DRIVER} \
+    --storage-driver ${DOCKER_STORAGE_DRIVER} \
     --exec-opt native.cgroupdriver=cgroupfs
+    $DOCKER_STORAGE_OPTS
```

Comment:
Should be $DOCKER_OPTS rather and `--storage-opt` and can be configured for different boards. If it's not set then it makes no harm.